### DISCUSSION TITLE
Just upgraded to Sequoia, and have a few quirks to deal with.

### DISCUSSION THREAD

**Discussion by ceez** [rxhanson/RectanglePro-Community#561] [Q&A]
Been using Rectangle Pro for about 3 years.     I just upgraded my MBP to Sequoia, and things are a bit confusing.

At first login, MacOS noted there was conflict between the built-in tiling and Rectangle Pro.  I opted to favor Rectangle, but I'm looking for a discussion, FAQ, whatever about the trade-offs.

I'm not even quite sure where everything is in Settings, now.    

Looking for observations from other that also upgraded

**Comment by rxhanson** [comment]
The macOS tiling settings are in Desktop & Dock, scroll down to Windows.

The macOS tiling provides snap areas for maximize on top, right half and left half on their respective sides, and quarters in the corners. The windows will also animate as they resize to that space, and the drag window resizing between those windows is animated. 

There's obviously a lot more in Rectangle Pro, but you could use the built-in stuff for some functionality and keep Rectangle Pro running for other functionality.

**Comment by ceez** [comment]
Thanks Ryan, I found the Windows settings shortly after I posted.

I also have KeyCue installed which helps me determine whether the HotKeys are active, but I haven't been able to get it to read Rectangle.   (I think it needs to be focused on an active window.

**Comment by chansearrington** [comment]
@rxhanson well... I chose the opposite option to check out the new built in features and now I don't know how to move back to Rectangle Pro.

How do I change from Built in Sequoia functionality, back to Rectangle Pro?

**Reply by rxhanson** [reply]
![image](https://github.com/user-attachments/assets/ba605c48-8a7a-4b55-b716-251298a06dba)
![image](https://github.com/user-attachments/assets/61173891-4e37-413f-9119-2254af6459ea)

**Reply by chansearrington** [reply]
THANK YOU!!!!!
